CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1967,6 +1967,10 @@ private void UpdateTooltip(string text)
             }
             else
             {
+                // hide old tooltip before showing the new one to update the position
+                if (toolTip.Content as string != text)
```

Comment:
Is the if statement even required? Is there a scenario where the same tooltip text is sent through?

Reply:
It is possible to get called when the tooltip was hidden because the mouse was over an area where no tooltip is shown. But in this case the tooltip.IsOpen is already false so no need to set it. But yeah, technically it can be set to IsOpen every time.